2022-07_sprint/sprint_parties/contributors.md

```diff
@@ -99,4 +99,26 @@ São Paulo, São Paulo (Brazil)
 [{fab}`github`](https://github.com/vitaliset)
 [{fab}`linkedin-in`](https://www.linkedin.com/in/carlo-lemos)
 :::
+:::{grid-item}
+
+<img alt="Image for contributor template" src="../../_static/contributors/blank_woman.jpg" class="rounded-circle">
+
+Mariya
+
+North America
+
+[{fab}`github`](https://github.com/mariyayb)
+[{fab}`linkedin-in`](https://www.linkedin.com/in/mariya-b)
+:::
+:::{grid-item}
+
+<img alt="Image for contributor template" src="../../_static/contributors/blank_man.jpg" class="rounded-circle">
+
```

Comment:
```suggestion
<img alt="Image for contributor template" src="../../_static/contributors/blank_man.jpg" 
 class="rounded-circle">

```

Reply:
@cluhmann I renamed the blank image from `png` extension to `jpg` extension.  Let's see if this works.